cmd/cluster-display/main.go

```diff
@@ -169,7 +169,7 @@ func main() {
 		interrupts.Terminate()
 	}
 
-	kubeConfigs, _, err := util.LoadKubeConfigs(o.hiveKubeconfigPath, kubeconfigChangedCallBack)
+	kubeConfigs, err := util.LoadKubeConfigs(o.hiveKubeconfigPath, "", kubeconfigChangedCallBack)
```

Comment:
Didn't you write the utility upstream with the express design of having composable options, so nobody ever passes `""` like this?